thoth/storages/graph/dgraph.py

```diff
@@ -763,6 +763,22 @@ def retrieve_unsolved_python_packages(self, solver_name: str) -> dict:
 
         return self._postprocess_retrieve_packages(result)
 
+    def retrieve_not_analyzed_python_package_version_entities(self) -> List[dict]:
```

Comment:
Should we rename differently?

Reply:
ah, good point... I would go with something shorter:

`retrieve_unanalyzed_python_package_versions`